README.rst

```diff
@@ -61,6 +61,7 @@ CLI Options
       num-task:                     number of task nodes
       release-label:                EMR release label
       s3-bucket:                    name of s3 bucket to upload spark file (required)
+      s3-path:                      path within s3-bucket to upload spark file (optional)
```

Comment:
I'd make the entire "source" prefix a CLI option with it defaulting `S3_KEY_PREFIX` instead of it hardcoded right now

Reply:
agreed, s3-path is now the root of where sparksteps writes `/logs` and `/sources`. Defaults to `/sparksteps`